README.md

```diff
@@ -13,6 +13,8 @@ $ source setup.sh
 $ source setup.sh --windows
 ```
 
+config.py is required to run and contains the Trello API tokens and Boar ID's
```

Comment:
This is no longer true. Just .env is needed, but with the correct env vars in there.

Can you also add instructions to the README for running tests and any prerequisites (like geckodriver)